### ISSUE TITLE
[Early Access] Enable UBP for unbilled teams whose owners are feature-flagged for UBP

### ISSUE THREAD

**Issue by jldec** [gitpod-io/gitpod#13894]
When individuals try out UBP for personal accounts via the "Early Access" program, they should also be able to try out UBP for their own team. This will allow a more complete evaluation and better feedback. 

Notes:

- These teams will not automatically receive a discount. 
- Team which already have Chargebee team-billing should not be auto-enabled for UBP until those plans are cancelled.

**Comment by geropl** [comment]
This adds a cyclic dependency. Let me think about how to unblock that.

**Comment by jankeromnes** [comment]
@jldec in your view, should the other team owners then also see Usage-Based (even though they were not feature-flagged at first)?

**Comment by geropl** [comment]
> in your view, should the other team owners then also see Usage-Based in their team (even though they were not feature-flagged at first)?

This already happens because all members inherit BillingMode `"usage-based"` if they are in at least one `"usage-based"` team (and don't have a personal Chargebee subscription), they would migrate automatically.

Let me play around with how this actually works in a preview env.

**Comment by jldec** [comment]
Closing this in favor of continuing manual team-level UBP feature-flags.
